### DISCUSSION TITLE
Local instance is greyed out

### DISCUSSION THREAD

**Discussion by nonetrix** [Jeffser/Alpaca#312] [Q&A] [answered]
I am unable to set it to use a local instance for some reason, it's just grey I haven't found how to bypass this. Seems to be using a embedded version which isn't playing nicely with my RX 6800 seemingly :/
![image](https://github.com/user-attachments/assets/fbd28b24-0e47-4029-b5ea-d02731f06f46)

**Comment by Jeffser** [comment]
Hi, you need to set the url first!

**Reply by nonetrix** [reply]
I did, it doesn't do anything

**Reply by Jeffser** [reply, marked as answer]
Did you press the check button that appears at the right of the textbox?

**Reply by nonetrix** [reply]
Are we looking at the same UI?! Where? All I see is a edit button. Regardless UI/UX needs improvement

**Reply by nonetrix** [reply]
Wait now it's there?! I swear it didn't appear for me yesterday??????????????????????????? But a better flow would be just to have it as a toggle, then it pops out the text fields with default local ip, and if nothing is there it just errors

**Reply by Jeffser** [reply]
Yeah you are right about that, it should just ask, I will consider it
